src/app/home/get.controller.js

```diff
@@ -11,8 +11,15 @@ module.exports = (req, res) => {
   const role = cookie.getUserRole();
   const orgId = cookie.getOrganisationId();
 
+  // Delete any GAR stored in the cookie session
+  cookie.session.gar = null;
+
   tokenApi.getLastLogin(cookie.getUserEmail())
     .then((userSession) => {
+      const { successHeader, successMsg } = req.session;
```

Comment:
GAR dashboard never had the logic in place to take the success message and display it, hence the point of this story.